sdk/analytics/src/main/java/com/klaviyo/analytics/Klaviyo.kt

```diff
@@ -47,6 +52,12 @@ object Klaviyo {
                 .applicationContext(applicationContext)
                 .build()
         )
+
+        val application = applicationContext.applicationContext as? Application
+        application?.apply {
+            unregisterActivityLifecycleCallbacks(Registry.lifecycleCallbacks)
+            registerActivityLifecycleCallbacks(Registry.lifecycleCallbacks)
+        } ?: throw LifecycleException()
```

Comment:
Would this crash their app? Should it log a warning instead?

Reply:
Yep! It will log a warning when #129 and #130 are merged

Reply:
oh nice!